apitest/src/test/java/bisq/apitest/method/RegisterDisputeAgentsTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.apitest.method;
+
+import bisq.proto.grpc.RegisterDisputeAgentRequest;
+
+import io.grpc.StatusRuntimeException;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import static bisq.apitest.config.BisqAppConfig.arbdaemon;
+import static bisq.common.app.DevEnv.DEV_PRIVILEGE_PRIV_KEY;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+
+
+@SuppressWarnings("ResultOfMethodCallIgnored")
+@Slf4j
+@TestMethodOrder(OrderAnnotation.class)
+public class RegisterDisputeAgentsTest extends MethodTest {
+
+    @BeforeAll
+    public static void setUp() {
+        try {
+            setUpScaffold("bitcoind,seednode,arbdaemon");
+        } catch (Exception ex) {
+            fail(ex);
+        }
+    }
+
+    @Test
+    @Order(1)
+    public void testRegisterArbitratorShouldThrowException() {
+        var req =
+                createRegisterDisputeAgentRequest("arbitrator");
+        Throwable exception = assertThrows(StatusRuntimeException.class, () ->
+                grpcStubs(arbdaemon).disputeAgentsService.registerDisputeAgent(req));
+        assertEquals("INVALID_ARGUMENT: arbitrators must be registered in a Bisq UI",
```

Comment:
Relying on string matching for tests makes them fragile. At a minimum we should probably add a note for strings that are used in this way.

Reply:
I will fix this in another PR.

Reply:
Yeah, I think this happens quite a bit in other tests as well iirc. Not urgent, but good to have a better solution for the future.

Reply:
This string matching problem has bitten me a few times, when I misspelled "bitciond" in some test case setups.  
PR [4540](https://github.com/bisq-network/bisq/pull/4540) fixes it.